docker/production/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  edpn-frontend-docker-production:
+    build:
+      context: ../../
```

Comment:
In the production runtime, I'd advise against this. The image should be built and pushed to a registry... and ops can pull it from there and run it.

If the _intent_ of this file is simply to build the Production image, no problem at all, but probably wants a separate runtime docker-compose.yml (or the build workflow can just point at the Dockerfile's).